src/integration/HubspotApiModel.ts

```diff
@@ -14,10 +13,14 @@ export class HubspotApiModel {
      * @param objectId objectID of contact to query
      * @param paramToGet query parameters to filter by. eg. 'email'.
      */
-    public async getContact(contactObjectID: number, paramToGet?: string){
-        console.log("this is current user ", get_current_user());
-        let token:any = await PollTokensFromDatabase(get_current_user());
+    public async getContact(contactObjectID: number,hub_id:number, paramToGet?: string){
+
+        console.log("this is current user ", hub_id);
```

Comment:
I think we should try to get away from saying 'current user' when this is the hubspot id. Maybe just say Hubspot ID or hubspot account id.